src/utils/transaction.ts

```diff
@@ -998,7 +1409,7 @@ export const estimateCrossChainAction = async (
     //
   }
 
-  return { gasCost, errorMessage, usdPrice, feeAmount };
+  return { gasCost, errorMessage, usdPrice, feeAmount, timeStamp };
```

Comment:
why `timestamp` is needed here on estimation type?

Reply:
you can reuse `createdTimestamp` where you need this, you would even have this prior to estimation, unless you need time specific estimations?